### PR TITLE
Feature/farm revamp

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -60,18 +60,20 @@ if (TAG) {
...
-    ignoreErrors: ['AbortError'],
-    integrations: [new BrowserTracing()],
-    tracesSampleRate: 0.1,
-  })
-
-  Sentry.configureScope(scope => {
-    scope.setTag('request_id', sentryRequestId)
-    scope.setTag('version', TAG)
-  })
+  if (window.location.href.includes('kyberswap.com')) {
```
Comment (namgold) [inline src/index.tsx:63]: Revert please. I fixed this

File: src/components/Header/index.tsx [hunk 2]
```diff
@@ -353,20 +353,15 @@ export default function Header() {
         <HeaderLinks>
           <HoverDropdown
             forceShowDropdown={isShowTutorial && stepInfo?.selector === `#${TutorialIds.BRIDGE_LINKS}`}
-            active={pathname.includes('/swap') || pathname === '/buy-crypto'}
+            active={pathname.includes('/swap') || pathname === '/buy-crypto' || pathname === '/bridge'}
```
Comment (namgold) [inline src/components/Header/index.tsx:356]: use `APP_PATHS.BRIDGE`

File: src/pages/IncreaseLiquidity/index.tsx [hunk 3]
```diff
@@ -65,9 +62,9 @@ export default function AddLiquidity({
 
   useEffect(() => {
     if (!!chainId && !!prevChainId && chainId !== prevChainId) {
-      history.push('/myPools')
+      navigate('/myPools')
```
Comment (hungdoansy) [inline src/pages/IncreaseLiquidity/index.tsx:65]: We can use APP_PATHS here

**Review by hungdoansy — CHANGES_REQUESTED** [review]

File: src/components/Tutorial/index.tsx [hunk 4]
```diff
@@ -148,11 +149,15 @@ function Tutorial({ customIcon, type }: Props) {
...
     <>
       {customIcon ? (
-        <div onClick={() => setShow(true)}>
-          <MouseoverTooltip text={t`Tutorial`} placement="top" width="fit-content">
-            {customIcon}
-          </MouseoverTooltip>
-        </div>
+        <Flex onClick={() => setShow(true)} alignItems="center">
+          {showTooltip ? (
+            <MouseoverTooltip text={t`Tutorial`} placement="top" width="fit-content">
```
Comment (nguyenhoaidanh) [inline src/components/Tutorial/index.tsx:154]: `<MouseoverTooltip disableTooltip={!showTooltip} ...`

Comment (viet-nv) [reply src/components/Tutorial/index.tsx]: done

File: src/pages/IncreaseLiquidity/index.tsx (hunk 3 above)
Comment (viet-nv) [reply src/pages/IncreaseLiquidity/index.tsx]: done

File: src/components/NavigationTabs/index.tsx [hunk 5]
```diff
@@ -115,21 +115,21 @@ const StyledMenuButton = styled.button<{ active?: boolean }>`
 export function SwapPoolTabs({ active }: { active: 'swap' | 'pool' }) {
   return (
     <Tabs style={{ marginBottom: '20px', display: 'none' }}>
-      <StyledNavLink id={`swap-nav-link`} to={'/swap'} isActive={() => active === 'swap'}>
+      <StyledNavLink id={`swap-nav-link`} to={'/swap'}>
```
[the remark this answers is not in the document]
Comment (viet-nv) [reply src/components/NavigationTabs/index.tsx]: done

File: src/components/Header/index.tsx [hunk 6]
```diff
@@ -427,26 +417,11 @@ export default function Header() {
                   <DropdownIcon />
                 </Flex>
                 <Dropdown>
-                  <StyledNavLink
-                    id="pools-nav-link"
-                    to="/pools"
-                    isActive={(match, { pathname }) => Boolean(match) || pathname.startsWith('/pools')}
-                    style={{ width: '100%' }}
-                  >
+                  <StyledNavLink id="pools-nav-link" to="/pools" style={{ width: '100%' }}>
```
[the remark this answers is not in the document]
Comment (viet-nv) [reply src/components/Header/index.tsx]: done

File: src/components/Header/index.tsx [hunk 7]
```diff
@@ -427,26 +417,11 @@ export default function Header() {
...
-                    to="/myPools"
-                    isActive={(match, { pathname }) =>
-                      Boolean(match) ||
-                      pathname.startsWith(APP_PATHS.CLASSIC_ADD_LIQ) ||
-                      pathname.startsWith(APP_PATHS.CLASSIC_REMOVE_POOL) ||
-                      pathname.startsWith(APP_PATHS.CLASSIC_CREATE_POOL) ||
-                      (pathname.startsWith(APP_PATHS.FIND_POOL) && pathname.endsWith(APP_PATHS.FIND_POOL))
-                    }
-                  >
+                  <StyledNavLink id="my-pools-nav-link" to="/myPools">
```
[the remark this answers is not in the document]
Comment (viet-nv) [reply src/components/Header/index.tsx]: done

File: src/components/ClassicElasticTab.tsx [hunk 8]
```diff
@@ -22,9 +22,10 @@ function ClassicElasticTab() {
   const notSupportedMsg = ELASTIC_NOT_SUPPORTED[chainId]
 
   const theme = useTheme()
-  const history = useHistory()
+  const location = useLocation()
+  const navigate = useNavigate()
 
-  const isFarmpage = history.location.pathname === '/farms'
+  const isFarmpage = location.pathname === '/farms'
```
[the remark this answers is not in the document]
Comment (viet-nv) [reply src/components/ClassicElasticTab.tsx]: done

File: src/components/ClassicElasticTab.tsx [hunk 9]
```diff
@@ -35,7 +36,7 @@ function ClassicElasticTab() {
             if (!!notSupportedMsg) return
             const newQs = { ...qs, tab: VERSION.ELASTIC }
             let type: MIXPANEL_TYPE | '' = ''
-            switch (history.location.pathname) {
+            switch (location.pathname) {
               case '/pools':
```
[the remark this answers is not in the document]
Comment (viet-nv) [reply src/components/ClassicElasticTab.tsx]: done

File: src/components/Header/index.tsx (hunk 2 above)
Comment (viet-nv) [reply src/components/Header/index.tsx]: done

File: src/components/Vesting/VestingCard.tsx [hunk 10]
```diff
@@ -65,18 +106,52 @@ const Dot = styled.div<{ color: string }>`
...
+  gap: 24px;
+  width: 100%;
+
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    flex-direction: column;
+    gap: 12px;
+  `}
+`
+
+const SummaryItem = styled.div<{ colorfull?: boolean }>`
```
Comment (hungdoansy) [inline src/components/Vesting/VestingCard.tsx:126]: typo: colorful

File: src/components/Vesting/styleds.tsx [hunk 11]
```diff
@@ -3,11 +3,11 @@ import styled from 'styled-components'
 export const ScheduleGrid = styled.div`
   display: grid;
   gap: 24px;
-  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-columns: 1fr;
   margin-top: 16px;
 
   ${({ theme }) => theme.mediaWidth.upToMedium`
```
Comment (hungdoansy) [inline src/components/Vesting/styleds.tsx:9]: upToMedium and upToSmall can be removed

File: src/components/YieldPools/ElasticFarmGroup/FarmCard.tsx [hunk 12]
```diff
@@ -0,0 +1,479 @@
...
+
+  transform: rotateY(${({ flip }) => (flip ? '-180deg' : '0')});
+`
+
+const FlipCardFront = styled.div`
+  display: flex;
+  height: 100%;
+  flex-direction: column;
+  backface-visibility: hidden;
+  -webkit-backface-visibility: hidden;
```
Comment (hungdoansy) [inline src/components/YieldPools/ElasticFarmGroup/FarmCard.tsx:57]: This line is not needed. SC will add the prefix automatically for us.

Comment (viet-nv) [reply src/components/YieldPools/ElasticFarmGroup/FarmCard.tsx:57]: updated.

File: src/components/Vesting/styleds.tsx (hunk 11 above)
Comment (viet-nv) [reply src/components/Vesting/styleds.tsx]: fixed

File: src/components/Vesting/VestingCard.tsx (hunk 10 above)
Comment (viet-nv) [reply src/components/Vesting/VestingCard.tsx]: fixed

**Review by hungdoansy — APPROVED** [review]

**Review by namgold — APPROVED** [review]

**Pull request closed without merging**